dm/dumpling/dumpling_test.go

```diff
@@ -146,21 +148,25 @@ func (t *testDumplingSuite) TestCallStatus(c *C) {
 	dumpConf.PromRegistry = tidbpromutil.NewDefaultRegistry()
 
 	s := m.Status(nil).(*pb.DumpStatus)
-	c.Assert(s.CompletedTables, Equals, float64(0))
-	c.Assert(s.FinishedBytes, Equals, float64(0))
-	c.Assert(s.FinishedRows, Equals, float64(0))
-	c.Assert(s.EstimateTotalRows, Equals, float64(0))
+	assert.Equal(t, s.CompletedTables, float64(0))
```

Comment:
How about use require.Equals, seems we don't use Assert in other test

Reply:
I have seen the difference between them. `Assert` doesn't exit the process, just print the diff log.

Reply:
https://github.com/pingcap/tiflow/pull/7511/commits/5bc9c850f20a9fc7e3f712f0ebc49543e22229e1